src/stability.md

```diff
@@ -72,6 +68,23 @@ even on an `unstable` function, if that function is called from another
 Furthermore this attribute is needed to mark an intrinsic as callable from
 `rustc_const_stable` functions.
 
+## Stabilizing a library feature
+
+To stabilize a feature, follow these steps:
+
+0. Ask a **@T-libs** member to start an FCP on the tracking issue and wait for
+   the FCP to complete (with `disposition-merge`).
+1. Change `#[unstable]` to `#[stable(since = "version")]`.
+   `version` should be the *current nightly*, i.e. stable+2. You can see which version is
+   currently nightly [on Forge](https://forge.rust-lang.org/#current-release-versions).
```

Comment:
I feel like `#[unstable]` might confuse people who haven't seen the unstable attribute before since it makes it look like it doesn't take arguments.

```suggestion
1. Change `#[unstable(...)]` to `#[stable(since = "version")]`.
   `version` should be the *current nightly*, i.e. stable+2. You can see which version is
   the current nightly [on Forge](https://forge.rust-lang.org/#current-release-versions).
```